proofs/signatures/th_lra_test.plf

```diff
@@ -11,9 +12,9 @@
   (@ p1 (polyc 2/1 m1)
   (@ p2 (polyc (~ 8/1) m2)
   (@ p3 (polyc 0/1 m3)
-  (% pf_nonneg_1 (th_holds (>=0_Real (poly_term p1)))
-  (% pf_nonneg_2 (th_holds (>=0_Real (poly_term p2)))
-  (% pf_nonneg_3 (th_holds (>=0_Real (poly_term p3)))
+  (% pf_nonneg_1 (th_holds (>=0_poly p1))
+  (% pf_nonneg_2 (th_holds (>=0_poly p2))
+  (% pf_nonneg_3 (th_holds (>=0_poly p3))
      (lra_contra_>=
```

Comment:
I think that currently in cvc4, it is usually crucial to know what the proof proves, and not only that it is a valid proof. If that is the case with LRA proofs as well, I would suggest including a `( : (holds ...)` in every test proof.

Reply:
You're right.
I got out of the habit when I was writing tests for `program`s, but I should get back into the habit.
I'll add the annotations.